gax-java/.github/release-please.yml

```diff
@@ -1,12 +0,0 @@
-releaseType: java-yoshi
-bumpMinorPreMajor: true
-handleGHRelease: true
-branches:
-  - releaseType: java-backport
-    bumpMinorPreMajor: true
-    handleGHRelease: true
-    branch: 2.16.x
-  - releaseType: java-backport
-    bumpMinorPreMajor: true
-    handleGHRelease: true
-    branch: 2.19.x
```

Comment:
How have we decided to handle these LTS branches for `gax-java`?

Reply:
Same as `api-common-java`, we will keep using the LTS branches in the old `gax-java` repo.